test/parallel/test-events-once.js

```diff
@@ -84,10 +84,72 @@ async function onceError() {
   strictEqual(ee.listenerCount('myevent'), 0);
 }
 
+async function onceWithEventTarget() {
+  const emitter = new class EventTargetLike extends EventEmitter {
```

Comment:
I'm not sure this is the right approach. I would prefer this not to extend EventEmitter.
Also, you can make it a top level helper.

Reply:
done.

Reply:
@mcollina I thought about implementing partial EventTarget and use it for testing.
Something like that [EventTarget Example](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget#Example) (Also, need to implement in that example above `once` functionality).
This approach seems to be a little bit exaggerated.

WDYT I should do here?
(If we decide to keep it like that, I will extract it to a top level helper)

Reply:
I'm -1 to have the `EventTarget` example extend `EventEmitter`. We do not need a full `EventTarget` implementation, but we could just mock/place assertions on the methods that are used by `once`.

Reply:
done.